### PR TITLE
Replace LightGraphs by Graphs, fix deprecations, update compat

### PR DESCRIPTION
**Pull request by greimel**
* LightGraphs has been renamed to Graphs (without change in functionality)
* there were two deprecation warnings in the tests that I fixed
* updated SpecialFunctions compat

I had to remove on test that relies on simulation/estimation. Since Travis CI has not run in a while I am not sure if this is an actual regression.

### CODE REVIEW

**Comment by greimel** [comment]
I moved CI from travis to GHA.

File: test/test_markov_approx.jl [hunk 1]
```diff
@@ -31,7 +31,7 @@
     X = simulate(mc, 100_000)
     mc2 = estimate_mc_discrete(X, mc.state_values)
     @test isequal(mc.state_values, mc2.state_values)
-    @test isapprox(mc.p, mc2.p; atol=1e-2)
+    @error norm(mc.p - mc2.p)
```
Comment (sglyon) [inline test/test_markov_approx.jl:34]: Can you explain what this change was for?

Thanks!

Comment (greimel) [reply test/test_markov_approx.jl:34]: This test is failing locally.(The norm of the difference is about 0.01)

But I don't think this is related to this PR.

Comment (greimel) [reply test/test_markov_approx.jl:34]: This test is also failing on the current release.

Comment (greimel) [reply test/test_markov_approx.jl]: @sglyon I reverted that line and removed the switch to Github Actions in this PR (this is now in #272)

Please let me know what I can do to facilitate the review for you.

**Comment by greimel** [comment]
rebased and closed/reopened to trigger CI

**Comment by greimel** [comment]
The failed test on latest and nightly is the same as before this PR

image

see #272

**Comment by greimel** [comment]
rebased again

**Comment by oyamad** [comment]
@greimel Thanks!
Let me merge this.

**Comment by greimel** [comment]
Thanks! Would you mind releasing a new version as well? (I've already incremented the version number)

**Pull request merged**